umh.docs.umh.app/content/en/docs/getstarted/dataacquisitionmanipulation.md

```diff
@@ -25,10 +25,29 @@ The UMH includes 3 pre-configured data simulators for testing connections:
 OPC UA, often complex, can be streamlined using our Benthos-based OPC UA connector
 accessible from the Management Console.
 
+### Visualize Network Devices or External Services
+
+In the **Network View**, you can visualize the network devices a UMH instance is connected to.
+This view provides the latest status of both the instances and the network devices, including
+external services.
+
+**Network devices** encompass physical devices with IP addresses as well as virtual devices that
+may exist in cloud environments or services accessed over the network, such as ERP systems with
+REST interfaces. **External services** refer to any services accessed externally, whether they
```

Comment:
First, let’s define what we mean by network devices and external services. A network device refers exclusively to any physical device located on the shop floor that is assigned an IP address. This includes machinery, sensors, and any hardware components that communicate over the network.

External services, on the other hand, encompass both virtual devices and services accessed beyond the local network. Virtual devices, such as software systems that have network capabilities but are not physically housed on the shop floor, fall into this category because they require network interaction but are not physical entities. This category also includes on-premises and cloud-hosted applications like ERP systems equipped with REST interfaces.